docs/release-template.md

```diff
@@ -74,7 +74,7 @@
 
 ## Redshift connector
 
-## SingleStore (MemSQL) connector
```

Comment:
Is this renamed in the connector docs as well? We will have to have some sort of "formerly known as MemSQL"

Reply:
It's at the top of the SingleStore page. I'm going to do a follow-up PR where I rename files and the page itself, though.

Reply:
I left "formerly known as MemSQL" intentionally so that people can find the page by "MemSQL" keyword. 

> I'm going to do a follow-up PR where I rename files and the page itself, though.

We don't support page redirection at this time. I'm not sure if renaming files is a good idea.